src/docfx/build/page/BuildPage.cs

```diff
@@ -176,6 +177,7 @@ private static SystemMetadata CreateSystemMetadata(ErrorBuilder errors, Context
 
             systemMetadata.SearchProduct = context.Config.Product;
             systemMetadata.SearchDocsetName = context.Config.Name;
+            systemMetadata.SearchEngine = context.Config.SearchEngine;
```

Comment:
what is `systemMetadata.SearchEngine` used for? seems like only `config.SearchEngine` is in use

Reply:
It is used by template to what search UI to show based on the search engine type,